README.md

```diff
@@ -31,14 +31,14 @@ On your project directory to build you project and push to desired registry.
 ##### Conditional or Optional parameters
 `--docker-registry` Docker registry to push to. This parameter will populate the `ACR_BUILD_DOCKER_REGISTRY` reserved environment variable (see `Build Environment`) Registry is required if `--push` options is present<br />
 `--docker-user` Username for the docker registry specified above<br />
-`--docker-secret` Password or token for registry specified above<br />
+`--docker-password` Password or token for registry specified above<br />
+`--web-archive` The URL of a tar.gz archive which contains the source code<br />
```

Comment:
>web [](start = 3, length = 3)

Lets rename this to --archive. 
I'm guessing you should be able to just merge --git-url and this to determine if its a clonable or just downloadable